pkg/clusteragent/admission/controllers/webhook/controller_v1beta1.go

```diff
@@ -242,3 +242,14 @@ func (c *ControllerV1beta1) getWebhookSkeleton(nameSuffix, path string) admiv1be
 
 	return webhook
 }
+
+func (c *ControllerV1beta1) getAdmiV1Beta1FailurePolicy() admiv1beta1.FailurePolicyType {
+	switch c.config.getFailurePolicy() {
+	case "Ignore":
+		return admiv1beta1.Ignore
+	case "Fail":
+		return admiv1beta1.Fail
```

Comment:
```suggestion
	case "Ignore", "ignore":
		return admiv1beta1.Ignore
	case "Fail", "fail":
		return admiv1beta1.Fail
```

Please add a test case for lower-case config 🙇